crates/stackable-webhook/src/lib.rs

```diff
@@ -173,6 +173,7 @@ impl WebhookServer {
     async fn run_server(self) -> Result<()> {
         debug!("run webhook server");
 
+        // TODO (@Techassi): Make opt-in configurable from the outside
```

Comment:
I would prefer we drop the concept of opt-in fields all together. We should just decide which additional fields are relevant, and record them.

Reply:
I'll mark as resolved, as it is covered here: https://github.com/stackabletech/operator-rs/pull/806#discussion_r1636282926